integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -154,8 +169,26 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
     protected Map<String, Object> getConnectConfigForCluster(String bootstrapServers) {
         Map<String, Object> kafkaConfig = new HashMap<>();
         String saslMechanism = getSaslMechanism();
+        String securityProtocol = getSecurityProtocol();
```

Comment:
I don't think this comes from this PR but I really don't like that `getSecurityProtocol()` returns `null`. 

I think it should return a default, probably `PLAINTEXT`, and ideally as the `SecurityProtocol` enum rather than dealing in strings.

Reply:
I agree using the enum instead of strings, but it is the only way I found to be sure that the test has not configured anything and take into account this use case as well.

Reply:
I'm not quite sure what you mean by not configuring anything, but as I understand it the default is `PLAINTEXT` there is no `null` option.